src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
```

Comment:
What's the point of these classes, I don't see you using it anywhere. 

Just introducing the class is not sufficient. These classes must be used in Address.java to make the code more meaningful.

Make the necessary changes to the Address.java class too.